pyanaconda/errors.py

```diff
@@ -116,6 +116,7 @@ def _get_default_mapping(self):
             InsightsClientMissingError.__name__: self._insightsErrorHandler,
             InsightsConnectError.__name__: self._insightsErrorHandler,
             "KickstartRegistrationError": self._kickstartRegistrationErrorHandler,
+            "SubscriptionTokenTransferError": self._subscriptionTokenTransferErrorHandler,
```

Comment:
You should use the `NonCriticalInstallationError` exception.

Reply:
I think that's a bigger change in code than whats in scope for this bug & right next to this error handler are others that could be quite likely converted as well and are definitely not in scope.

Still, I think we could certainly use the new mechanism for the port to Fedora.